src/main/kotlin/com/bada/app/models/AbstractEntity.kt

```diff
@@ -26,4 +26,4 @@ abstract class AbstractEntity<T : Serializable>(
     }
 }
 
-abstract class AbstractEntityLong : AbstractEntity<Long>(-1)
\ No newline at end of file
+abstract class AbstractEntityLong : AbstractEntity<Long>()
```

Comment:
Please remove `AbstractEntityLong` and replace it's reference with the generic type.